models/sandbox_endpoint.go

```diff
@@ -35,6 +35,9 @@ type SandboxEndpoint struct {
 	// route type
 	RouteType string `json:"routeType,omitempty"`
 
+	// target
```

Comment:
I think this model itself needs to be renamed `Endpoint` since it's not tied to a sandbox anymore

Reply:
Actually, standalone RGs are using a different endpoint struct (https://github.com/signadot/signadot/blob/master/workspaces/app/api/signadotv2/routegroups/routegroup.go#L22). We should probably converge these to just one.